Update pcs.py - Different roles

maxpascher opened the pull request:
Check the PCS user's role to generate the correct download link for the spreadsheet.

maxpascher committed:
Update pcs.py - Different roles

Check the PCS user's role to generate the correct download link for the spreadsheet.

maxpascher commented:
@sven-mayer, @RaphaelWimmer , @sigchi-ops

maxpascher committed:
Update taps.py

In order to make the tools compatible with TAPS again, taps.py was revised. A few functions were adjusted and a PROC_ID was added.

maxpascher commented:
In order to make the tools compatible with TAPS again, taps.py was revised. A few functions were adjusted and a PROC_ID was added.

RaphaelWimmer commented:
I'm happy that this heap of code is still of use. Thank you for updating it!

I'm not actively involved with any SIGCHI publications at the moment, so I'm rather hesitant to merge pull requests - maybe someone else feels responsible (@sven-mayer ?).

In addition, I think the code in the first commit might be slightly improved:

```
for role in roles:
        match = re.match(r'<a href="/(\w+)/(\w+)">(.+)</a>', role[3])
        track_id_check = match.group(1)
        role_id_check = match.group(2)
        if role_id_check in ['pubchair', 'chair']:
            available_tracks[track_id_check] = role_id_check
role_id = available_tracks[track_id]
   
if role:
        r = pcs_session.get(PCS_SPREADSHEET_URL_PREFIX + track_id + "/" + role_id + PCS_SPREADSHEET_URL_SUFFIX)
```

I think the `if role` should actually be `if role_id`, right?

maxpascher committed:
Update rename_files.py

Third argument to set a prefix (e.g., "chi23b-") in the file names

maxpascher committed:
Update pcs.py

Fixed small issue in checking access rights for track.

When generating the list of tracks, removed 'access check' because it is

maxpascher commented:
Hey @RaphaelWimmer,

thanks. 
Yes, you are right, `if role_id` would is correct and I fixed it.
